src/Compilers/CSharp/Test/Symbol/Symbols/MissingSpecialMember.cs

```diff
@@ -990,10 +991,10 @@ public void AllWellKnownTypeMembers()
                     case WellKnownMember.System_Runtime_CompilerServices_IsUnmanagedAttribute__ctor:
                     case WellKnownMember.System_Runtime_CompilerServices_ITuple__get_Item:
                     case WellKnownMember.System_Runtime_CompilerServices_ITuple__get_Length:
+                    case WellKnownMember.System_GC__AllocateUninitializedArray_T:
                         // Not always available.
                         continue;
                 }
-                if (wkm == WellKnownMember.Count) continue; // Not a real value.
```

Comment:
@333fred Is it expected the test pass without skipping `WellKnownMember.Count`. That looks surprising to me.

Reply:
See line 915: this `continue` is dead and will never be reached.

Reply:
Yes this isn't related to my change at all I just noticed this statement was redundant

Reply:
or yeah sorry unreachable I should say